docs/user/tutorials/01-90-set-asynchronous-connection.md

```diff
@@ -54,7 +55,9 @@ This tutorial shows only one possible use case. There are many more use cases on
     ```
 
 3. Provide your Function logic in the `handler.js` file:
->**NOTE:** In this example, there's no sanitization logic. The `sanitize` Function is just a placeholder.
+
```

Comment:
This note + code block + text are not indented correctly

Reply:
Fixed, I think :D